scripts/internal_links_monitor.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python2.7
+# -*- coding: utf-8 -*-
+
+import syslog
+
+
+from swsssdk import  SonicV2Connector
+from sonic_py_common import multi_asic
+
+# Global variables
+MIN_SCAN_INTERVAL = 60      # Every minute
+MAX_SCAN_INTERVAL = 3600    # An hour
```

Comment:
how they are getting used ?

Reply:
not used in 201911, it was added for master PR. I can remove it from here.